### PR TITLE
EvaluationNavigationBar Component

### PR DESCRIPTION
**Pull request by abhishekpratapa**
Components
- [x] EvaluationTopBar
- [x] EvaluationNavigationBar
- [ ] EvaluationScores
- [ ] EvaluationScoresConsiderations
- [ ] EvaluationSummary
- [ ] EvaluationLabelMenu
- [ ] EvaluationLabelStats
- [ ] EvaluationIncorrectlyClassified
- [ ] EvaluationImageComponent
- [ ] EvaluationConfusionMenu

- Added a react component for the navigation bar component of evaluate.

### CODE REVIEW

**Comment by znation** [comment]
@abhishekpratapa I don't see any changes in this PR. Did you mean to add some new files to the commit?

**Comment by abhishekpratapa** [comment]
> @abhishekpratapa I don't see any changes in this PR. Did you mean to add some new files to the commit?

Forgot a `git commit` thanks for the catch!

File: src/visualization/Turi Create Visualization.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -22,6 +22,14 @@
 		FCB8DFE3206F1756001A7089 /* index.css in Resources */ = {isa = PBXBuildFile; fileRef = FCB8DFE2206F1756001A7089 /* index.css */; };
 		FCB8DFE5206F187D001A7089 /* index.js in Resources */ = {isa = PBXBuildFile; fileRef = FCB8DFE4206F187D001A7089 /* index.js */; };
 		FCE3954420B4C2CB0029FDB5 /* build in Resources */ = {isa = PBXBuildFile; fileRef = FCE3954320B4C2CB0029FDB5 /* build */; };
+		FCE91BE721E3CE1B00EFA3B5 /* index.scss in Resources */ = {isa = PBXBuildFile; fileRef = FCE91BDE21E3CE1B00EFA3B5 /* index.scss */; };
+		FCE91BE821E3CE1B00EFA3B5 /* index.js in Resources */ = {isa = PBXBuildFile; fileRef = FCE91BDF21E3CE1B00EFA3B5 /* index.js */; };
+		FCE91BE921E3CE1B00EFA3B5 /* selected_list_view.png in Resources */ = {isa = PBXBuildFile; fileRef = FCE91BE121E3CE1B00EFA3B5 /* selected_list_view.png */; };
```
Comment (znation) [inline src/visualization/Turi Create Visualization.xcodeproj/project.pbxproj]: Use SVG for image assets where possible.

**Comment by znation** [comment]
One comment, otherwise looks good!

**Pull request closed without merging**